src/js/view/view.js

```diff
@@ -777,6 +782,13 @@ define([
             if (!provider) {
                 return;
             }
+
+            if (_lastMediaWidth === width && _lastMediaHeight === height) {
```

Comment:
If width or height are percentages, then we must not exit. stretching must be recalculated.